File: code/cs/OpcClient.Classic.cs
// DOC
using Opc.UaFx.Client;
using Opc.UaFx.Client.Classic;

// DOC
var client = new OpcClient("opc.com://localhost:48410/<progId>/<classId>");

// DOC
client.Connect();

// DOC
// Your code to interact with the server.

// DOC
client.Disconnect();

// DOC
using (var client = new OpcClient("opc.com://localhost:48410/<progId>/<classId>")) {
    client.Connect();
    // Your code to interact with the server.
}

File: code/vb/OpcClient.Classic.vb
'DOC
Imports Opc.UaFx.Client
Imports Opc.UaFx.Client.Classic

'DOC
Dim client As New OpcClient("opc.com://localhost:48410/<progId>/<classId>")

'DOC
client.Connect()

'DOC
'Your code to interact with the server.

'DOC
client.Disconnect()

'DOC
Using client As New OpcClient("opc.com://localhost:48410/<progId>/<classId>")
    client.Connect()
    'Your code to interact with the server.
End Using
